apps/src/applab/applab.js

```diff
@@ -124,7 +124,8 @@ var twitterOptions = {
 };
 
 function stepDelayFromStepSpeed(stepSpeed) {
-  return 300 * Math.pow(1 - stepSpeed, 2);
+  // 3 sec per block in turtle mode
+  return 1500 * Math.pow(1 - stepSpeed, 2);
```

Comment:
If the unit is milliseconds, and stepSpeed is 0 for the most turtle mode, wouldn't this return 1.5 seconds, rather than 3?

Reply:
That is so true! I believe the reason why I said it's 3 seconds per block is because when the block has `x = 10` then `10` is highlighted with yellow for 1.5 seconds and then `x = 10` as a whole block is highlighted again for 1.5 seconds so totaling 3 seconds for the entire block.

But it makes more sense to be written as 1.5 seconds per socket so I'll go ahead and make the change. Thanks for pointing that out!